### PR TITLE
Us 61 task 67 74

### PR DESCRIPTION
**Pull request by Kanchan-Wakchaure**
1. Sign up UI and back end functionality
2. Refactoring and restructuring of all UI files
3. Navigation to different screens
4. Header and Footer components

### CODE REVIEW

File: Travlendar/frontend/src/components/login/Login.js [hunk 1]
```diff
@@ -110,7 +108,7 @@ class Login extends React.Component{
                                         color="primary"
                                         className="submit"
                                         onClick={this.handleSubmit}>
-                                    Login
+                                    Sign Up
```
Comment (namrathaov) [inline Travlendar/frontend/src/components/login/Login.js]: Change to login

File: Travlendar/users/views.py [hunk 2]
```diff
@@ -3,18 +3,20 @@
...
 from django.contrib.auth import authenticate
-
 from .forms import CustomUserCreationForm
 
 @api_view(['GET','POST'])
 def signUp(request):
     if request.method == 'POST':
-        form=CustomUserCreationForm(request.POST)
+        form=CustomUserCreationForm(request.data)
+        print("form data",request.data)
```
Comment (namrathaov) [inline Travlendar/users/views.py]: remove

File: Travlendar/frontend/src/components/footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  );
+}
+
+class Footer extends React.Component{
+    render(){
+        return(
+            <footer className="footer" style={{backgroundColor: '#3F51B5'}}>
+                <Container maxWidth="lg">
+                    <Typography variant="h6" align="center" gutterBottom>
+                        <b>Travlendar</b>
```
Comment (namrathaov) [inline Travlendar/frontend/src/components/footer/Footer.js:33]: Just a note for future: Change text color to white to match header.

**Review by namrathaov — CHANGES_REQUESTED** [review]
All the navigation and integration works great! Looks good to be merged with minor changes reviewed.

File: Travlendar/frontend/src/components/footer/Footer.js (hunk 3 above)
Comment (Kanchan-Wakchaure) [reply Travlendar/frontend/src/components/footer/Footer.js:33]: Done

File: Travlendar/users/views.py (hunk 2 above)
Comment (Kanchan-Wakchaure) [reply Travlendar/users/views.py]: Done

File: Travlendar/frontend/src/components/login/Login.js (hunk 1 above)
Comment (Kanchan-Wakchaure) [reply Travlendar/frontend/src/components/login/Login.js]: Done

**Comment by Kanchan-Wakchaure** [comment]
Added required changes.

**Review by namrathaov — APPROVED** [review]
Looks great to be merged now!

**Review by gaddevmounika — APPROVED** [review]
Navigation working fine.Home page looks good.Can be merged.
Note for future : buttons on Header must change dynamically when the user logs in.

**Pull request merged**